pkg/controller/controllerutil/controllerutil.go

```diff
@@ -116,18 +116,26 @@ const ( // They should complete the sentence "Deployment default/foo has been ..
 
 // CreateOrUpdate creates or updates the given object obj in the Kubernetes
 // cluster. The object's desired state should be reconciled with the existing
-// state using the passed in ReconcileFn. obj must be a struct pointer so that
+// state using the passed in MutateFn. obj is just a struct pointer so that
```

Comment:
nit: docs are wrong now (nothing is passed to mutate function, the sentence about `obj` needs to be updated too)